src/common/tests/utest_common.c

```diff
@@ -38,6 +38,13 @@ struct utest_root {
 	uint64_t	ur_root[0];
 };
 
+struct utest_mem_status {
```

Comment:
Move to utest_context so no need to track separate thing.

Reply:
Done.